docs/pages/application-access/guides/connecting-apps.mdx

```diff
@@ -124,7 +73,25 @@ An application name should make a valid sub-domain (\<=63 characters, no spaces,
 After Teleport is running, users can access the app at `app-name.proxy_public_addr.com`
 e.g. `grafana.teleport.example.com`. You can also override `public_addr` e.g
 `grafana.acme.com` if you configure the appropriate DNS entry to point to the
-Teleport proxy server.
+Teleport proxy server for self-hosted Teleport instances.
+
+<Admonition type="tip" title="Application URI Domains and TLS for Self-Hosted" scope="oss,enterprise" scopeOnly>
+Subdomains (*.teleport.example.com) and any other FQDN used as application URIs should have their entries
```

Comment:
We can use [this partial](https://github.com/gravitational/teleport/blob/9b843bad0c29c1b26587c2a3a1245d935e66633e/docs/pages/includes/dns-app-access.mdx) instead of this opening paragraph.

Also, we shouldn't need to mention Teleport Cloud if this text only appears for self-hosted users.

Finally, I think we need some kind of transition sentence to introduce the configuration snippet below.

Reply:
@ptgott addressed